eox_core/api/v1/views.py

```diff
@@ -56,16 +55,27 @@ def post(self, request, *args, **kwargs):
         """
         Creates the users on edxapp
         """
-        serializer = EdxappCourseEnrollmentQuerySerializer(data=request.POST)
+        multiple_responses = []
+        many = isinstance(request.data, list)
+        serializer = EdxappCourseEnrollmentQuerySerializer(data=request.data, many=many)
         serializer.is_valid(raise_exception=True)
+        data = serializer.validated_data
+        if not isinstance(data, list):
+            data = [data]
+
+        for one in data:
+            enrollment, msg = create_enrollment(**one)
+            serialized_enrollment = EdxappCourseEnrollmentSerializer(enrollment)
+            response_data = serialized_enrollment.data
+            if msg:
+                response_data["messages"] = msg
+            multiple_responses.append(response_data)
+
+        if many:
+            return Response(multiple_responses)
```

Comment:
Just a reminder of the discussion we had per other channel to make this a bit cleaner

Reply:
@felipemontoya Should I make the change that we discussed here (the decorator) or is that for a later refactor?

Reply:
If you can manage to do that in a few hours tops, you can add it here. If not, then merge now and we will use this code as it is for a few years ;)